LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-color-001.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
```

Comment:
Thought I'd add scrollbar-color tests as well as scrollbar-width for the sake of completeness.